app/routes/signup.js

```diff
@@ -13,16 +15,32 @@ export default class SignupRoute extends Route {
     });
     const userData = await response.json();
     if (response.status === 401) {
-      alert('You have not logged in. Please login first to fill this form.');
-      window.open(
-        'https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97',
-        '_self'
+      this.toast.warning(
```

Comment:
401 comes under the error category. Can we please use error instead of warning here?